### PR TITLE
Added sentiment capabilities for the Twilio app.

### PR DESCRIPTION
**Pull request by juliaferraioli**
Twilio will now send back sentiments that have a rating of "very likely", "likely", and "possible", in aggregate over all the faces it detects.

### CODE REVIEW

**Comment by amygdala** [comment]
I think the CLA issue is either that your google address isn't associated with your github account, or if it is, try making it the 'primary' one.

File: python/twilio-labels/whats_that.py [hunk 1]
```diff
@@ -92,10 +110,28 @@ def construct_message(labels):
...
+    return joy, anger, sorrow, surprise
 
+
+def get_labels(image, num_retries=3, max_labels=3, max_faces=10):
+    """Given an image, execute the label request"""
     labels = ""
+    face_annotations = ""
 
     # Set up the service that can access the API
     http = httplib2.Http()
```
Comment (amygdala) [inline python/twilio-labels/whats_that.py]: You should be able to simplify this bit -- see e.g. https://github.com/GoogleCloudPlatform/cloud-vision/blob/master/python/label/label.py#L46

**Comment by amygdala** [comment]
That LGTM aside from that little change on the credentials stuff. Cool stuff.

I had one high-level question -- if there is a photo with a bunch of people in it, the response might be too much text to be easily readable on a mobile device. Did you find that to be the case?  Wondering if it could make sense to truncate the face info in some way if it is 'too long'-- returning the 'very likely' readings first, but maybe dropping the less likely ones.

**Comment by amygdala** [comment]
I think these changes have now been subsumed by https://github.com/GoogleCloudPlatform/cloud-vision/pull/70, meaning that you can abandon this PR. (Take a look when you get a chance and LMK if there's something I missed).

**Comment by juliaferraioli** [comment]
Argh, sorry for not getting this cleaned up sooner. Yep, LGTM.

**Pull request closed without merging**